src/components/app-menu/AppMenu.tsx

```diff
@@ -1,38 +1,41 @@
-import { FC } from 'react'
+import Menu from '~/design-system/components/menu/Menu'
+import { MenuItemColorVariant } from '~/design-system/components/menu-item/MenuItem.constants'
 
-import Menu, { MenuProps } from '@mui/material/Menu'
-
-import { spliceSx } from '~/utils/helper-functions'
-
-import { PositionEnum } from '~/types'
-import { styles } from '~/components/app-menu/AppMenu.styles'
-
-interface AppMenuProps extends MenuProps {
+interface AppMenuProps {
+  anchorEl: HTMLElement | null
+  menuList: {
+    title: string
```

Comment:
```suggestion
    title: string | ReactNode
```
Maybe with this change you can delete `as unknown as string` in `NotificationsMenu.tsx`, but i am not sure